.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,4 +12,3 @@ StackOverflow discussion that's relevant]
 - [ ] New functionality includes tests
 - [ ] All tests pass
 - [ ] RELEASE\_NOTES.md has been updated if required (not required for bugfixes, required for API changes)
-- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>
```

Comment:
This was more just a reminder to users since people miss it a lot

Reply:
I'd be in favor of having this match across chef, chefdk, chefws.